packages/server/src/api/controllers/table/utils.ts

```diff
@@ -15,7 +15,7 @@ import { getViews, saveView } from "../view/utils"
 import viewTemplate from "../view/viewBuilder"
 import { cloneDeep } from "lodash/fp"
 import { quotas } from "@budibase/pro"
-import { events, context } from "@budibase/backend-core"
+import { events, context, db } from "@budibase/backend-core"
```

Comment:
Need to rename this - but for safety can we import this as `db as dbCore` - try to reserve the `db` keyword to avoid overlap!

Reply:
Totally understand this. Is it worth, in a different PR, changing this export to be called dbCore so you never have to rename it like this?

Reply:
We probably could do that - we'd just need to go through everywhere its imported and update it to remove the as!